src/customer/customer-strategy-selector.ts

```diff
@@ -7,10 +7,12 @@ import CustomerStrategyState, { DEFAULT_STATE } from './customer-strategy-state'
 export default interface CustomerStrategySelector {
     getSignInError(methodId?: string): Error | undefined;
     getSignOutError(methodId?: string): Error | undefined;
+    getExecutePaymentMethodCheckoutError(methodId?: string): Error | undefined;
```

Comment:
getExecutePaymentMethodCheckoutError(methodId?: string): Error | undefined;
```suggestion
    getExecutePaymentMethodCheckoutError?(methodId?: string)?: Error | undefined;
```

Reply:
Do I need to fix all getError methods in the file? Or can I live it right how it is to match the stile of the file?

Reply:
That will be nice you can 👍

Reply:
I can't do that :(
Because when I try to change the code `getExecutePaymentMethodCheckoutError?(methodId?: string)?: Error;` TSLint returns an error (see screenshot bellow)

<img width="747" alt="Screenshot 2021-08-20 at 13 18 18" src="https://user-images.githubusercontent.com/25133454/130219043-e09d6bb7-8573-433a-aac3-053ec0cf057f.png">

Reply:
Sorry was a typo, supposed to be 
`    getExecutePaymentMethodCheckoutError?(methodId?: string): Error;
`

Reply:
I think, that the reason why we should left it (`getExecutePaymentMethodCheckoutError(methodId?: string): Error | undefined;`) that we can have undefined as a result of the function call. Is it right?